blaze/expr/strings.py

```diff
@@ -83,6 +110,66 @@ class str_lower(UnaryStringFunction):
     def schema(self):
         return self._child.schema
 
+class str_isalnum(UnaryStringFunction): schema = bool_
+class str_isalpha(UnaryStringFunction): schema = bool_
+class str_isdecimal(UnaryStringFunction): schema = bool_
+class str_isdigit(UnaryStringFunction): schema = bool_
+class str_islower(UnaryStringFunction): schema = bool_
+class str_isnumeric(UnaryStringFunction): schema = bool_
+class str_isspace(UnaryStringFunction): schema = bool_
+class str_istitle(UnaryStringFunction): schema = bool_
+class str_isupper(UnaryStringFunction): schema = bool_
+
+class StrFind(ElemWise):
+    """
+    Find literal substring in string column.
+
+    """
+
+    __slots__ = '_hash', '_child', 'sub'
+    schema = datashape.Option(datashape.int64)
+
+
+@copydoc(StrFind)
+def str_find(col, sub):
+    if not isinstance(sub, basestring):
+        raise TypeError("'sub' argument must be a String")
+    return StrFind(col, sub)
+
+class Replace(ElemWise):
+    __slots__ = '_hash', '_child', 'old', 'new', 'max'
+    schema = datashape.string
```

Comment:
Isn't `replace` unconditionally returning a string ? I thought `datashape.Option()` was only required if the result may be `None`...